terraform/modules/hono/variables.tf

```diff
@@ -207,3 +207,15 @@ variable "grafana_dns_name" {
   type        = string
   description = "Name of the DNS host for Grafana"
 }
+
+variable "enhanced_mqtt_load_balancer" {
```

Comment:
Rename to advanced_load_ablancer - when moved under mqtt_adapter object

Reply:
done